components/Navigation/index.js

```diff
@@ -5,12 +5,21 @@ import { useRouter } from "next/router";
 
 export default function Navigation() {
   const router = useRouter();
+
+  const handleHomeClick = () => {
+    if (router.pathname === "/") {
+      router.reload();
+    } else {
+      router.push("/");
+    }
+  };
```

Comment:
@onemanwenttomow thanks for reviewing our code and the great recommendations! 🙌 

The reason we added the reloading on the home icon is because we thought there might be a scenario where the user wants to go back to the home page while typing in a search request and doesn't want to delete the typed in letters to go back to the actual list of Outivities. To "avoid" this, the user can press the home button.

I guess instead we could also add an "x" in the search bar to delete the request, but for now we decided to solve it this way.